react-exercises/pure-html-react/form-validation/fruit-pattern.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Fruit Pattern</title>
+</head>
+
+<body>
+  <form action="" method="post">
+    <label for="fruit-length">Would you prefer a banana or a cherry?</label>
+    <input type="text" name="fruit-length" pattern="[A-Za-z]{6,6}">
```

Comment:
Here expected input is either `banana` or `cherry` so `pattern="banana|cherry"` is much more appropiate.